### PR TITLE
Chat box

### LINKED ISSUE #28: Chat room button dropdown

### PR DESCRIPTION
**Pull request by gabremo**
Implementing a more concrete layout for the chat textbox

### CODE REVIEW

File: capstone-project/ui/src/components/ChatRoom/ChatRoom.jsx [hunk 1]
```diff
@@ -1,62 +1,84 @@
...
-            bName="chat button open";
-        }*/
-        console.log("Actual classname of chat-container", cName)
-      }, [chatOpen, handleToggle])
-
+    else{
+        cName="chat-container open";
+        bName="chat open";
+    }
+    console.log("Actual classname of chat-container", cName)
```
Comment (LifeBringer) [inline capstone-project/ui/src/components/ChatRoom/ChatRoom.jsx]: Rogue console.log

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Console.log found

**Comment by LifeBringer** [comment]
Please fill the panel on the right 'Development' 
image

File: capstone-project/ui/src/components/ChatRoom/ChatRoom.jsx (hunk 1 above)
Comment (gabremo) [reply capstone-project/ui/src/components/ChatRoom/ChatRoom.jsx]: Thanks for pointing it out!

**Review by Rebecca-CYLi — APPROVED** [review]
revisions fixed, G2G

**Pull request merged**